src/Workspaces/Core/Portable/Workspace/Solution/Project.cs

```diff
@@ -100,15 +100,15 @@ internal Project(Solution solution, ProjectState projectState)
         /// The language services from the host environment associated with this project's language.
         /// </summary>
         [Obsolete($"Use {nameof(Services)} instead.")]
-        public HostLanguageServices LanguageServices => _projectState.LanguageServices;
+#pragma warning disable CS0612 // warning CS0612: 'LanguageServices.HostLanguageServices' is obsolete
```

Comment:
Why did we need the pragma here? I thought calling Obsolete code from Obsolete code was fine?